### PR TITLE
[Test] Large scale dask on ray test

### PR DESCRIPTION
**Pull request by rkooo567**
## Why are these changes needed?

Add a large scale dask on ray workload script to the release pipeline. 

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by rkooo567** [comment]
cc @jennakwon06

File: release/data_processing_tests/README.rst [hunk 1]
```diff
@@ -1,9 +1,27 @@
...
+
+This process is required because ulimit is not permitted for images that we are using. Ulimit is necessary for large cluster testing like this.
+Check out https://discuss.ray.io/t/setting-ulimits-on-ec2-instances/590/2 for more details
+
+Success Criteria
+----------------
+
+For `streaming_shuffle.py`, make sure to include the output string to the release logs.
+
+For `dask_on_ray_large_scale_test.py`, make sure the test runs for at least for an hour. This test should succeed, otherwise, it is a release blocker.
```
Comment (ericl) [inline release/data_processing_tests/README.rst:27]: How long is it supposed to take?

**Review by ericl — APPROVED** [review]

File: release/data_processing_tests/README.rst (hunk 1 above)
Comment (ericl) [reply release/data_processing_tests/README.rst:27]: (why do we need to check it run more than an hour?)

Comment (rkooo567) [reply release/data_processing_tests/README.rst:27]: I will add more reasoning to the doc. But basically, I use the same parameter as they are using it for their production, and it is supposed to last many days with that param.

For now, I just want to make sure the initial load won't crash the job & cluster, so I require it to run for 1 hour (it is also very expensive. It costs around $500 per hour). We should iterate on this and have more strict conditions later (e.g., update the script to produce some output that can be comparable).

**Pull request merged**